superset/config.py

```diff
@@ -682,6 +682,11 @@ class D3Format(TypedDict, total=False):
 SCREENSHOT_WAIT_FOR_ERROR_MODAL_VISIBLE = 5
 # Max time to wait for error message modal to close, in seconds
 SCREENSHOT_WAIT_FOR_ERROR_MODAL_INVISIBLE = 5
+# event that Playwright waits for when loading a new page
+# possible values: "load", "commit", "domcontentloaded", "networkidle"
```

Comment:
Could we maybe add a link to the docs where these options are described? https://playwright.dev/docs/api/class-page#page-go-back-option-wait-until

Reply:
Good point, added!